src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateConverter.cs

```diff
@@ -14,8 +14,66 @@
 
 namespace GitHub.DistributedTask.Pipelines.ObjectTemplating
 {
-    internal static class PipelineTemplateConverter
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class PipelineTemplateConverter
     {
+        public static List<Step> ConvertToSteps(
```

Comment:
most of this method is existing, i just moved it up because it's public now

Reply:
is this already on server-side code or you need to copy it over?

Reply:
i need to make the same change on the server side. Waiting on until we have telemetry first to make sure we can use `__` to indicate generated